{
  "path": "/documentation/xctest/xctest/3521150-setupwitherror",
  "type": "Instance Method",
  "name": "setUpWithError()",
  "desc": "Provides an opportunity to reset state and to throw errors before calling each test method in a test case.",
  "items": [],
  "declaration": "func setUpWithError() throws"
}